docs/products/opensearch/howto/connect-with-python.rst

```diff
@@ -0,0 +1,47 @@
+Connect to OpenSearch cluster with Python
+=========================================
+
+One nice way to interact with your cluster is by using the `Python OpenSearch client <https://github.com/opensearch-project/opensearch-py>`_. 
```

Comment:
I wonder if it was me who used word "nice" initially in the tutorial :D But let's remove it and make a neutral  statement. For example "You can interact with your OpenSearch cluster with the help of ...."